app/templates/index.hbs

```diff
@@ -1,10 +1,7 @@
-{{! template-lint-disable no-redundant-role }}
 <div class="bg-shape-boxes pt-5" local-class="container">
   <div class="mb-4 pt-4" local-class="header-container">
     <img
-      alt=""
-      role="presentation"
-      src="../images/ember-help-wanted-logo.svg"
+      alt="help wanted" src="../images/ember-help-wanted-logo.svg"
```

Comment:
This should stay as role presentation, I think the rule was `no-redundant-role` because we had an empty alt and a `role=presentation` but we only need one of those two things

Reply:
breaks if we add `role="presentation"` 
<img width="937" alt="image" src="https://github.com/user-attachments/assets/59eec401-e55a-4403-9771-2b3ea2bf6a50" />

Reply:
![Screenshot 2025-03-17 at 12 06 02 PM](https://github.com/user-attachments/assets/1f7367b2-cb15-4ddd-bf77-031d9ec767c2)
@MehulKChaudhari @mansona FWIW this is what co-pilot was suggesting.

Reply:
Lets keep it alt=""?